packages/perseus/src/util/graphie.ts

```diff
@@ -690,21 +960,164 @@ export class Graphie {
+                $span.processText(text);
+            }
+
+            return $span;
+        });
+    };
 
     plotParametric(
         fn: (t: number) => Coord,
         range: Interval,
         style?: Record<string, any>,
-    ): RaphaelElement {}
+    ): RaphaelElement {
+        return this.withStyle(style, () => {
+            // We truncate to 500,000, since anything bigger causes
+            // overflow in the firefox svg renderer.  This is safe
+            // since 500,000 is outside the viewport anyway.  We
+            // write these functions the way we do to handle undefined.
+            const clip = (xy) => {
+                if (Math.abs(xy[1]) > 500000) {
+                    return [xy[0], Math.min(Math.max(xy[1], -500000), 500000)];
+                }
+                return xy;
+            };
+            const clippedFn = (x) => clip(fn(x));
+
+            if (!this.currentStyle.strokeLinejoin) {
+                this.currentStyle.strokeLinejoin = "round";
+            }
+            if (!this.currentStyle.strokeLinecap) {
+                this.currentStyle.strokeLinecap = "round";
+            }
+
+            const min = range[0];
+            const max = range[1];
+            let step = (max - min) / (this.currentStyle["plot-points"] || 800);
+            if (step === 0) {
+                step = 1;
+            }
+
+            const paths = this.raphael.set();
+            let points = [];
+            let lastY = clippedFn(min)[1];
+
+            for (let t = min; t <= max; t += step) {
+                const point = clippedFn(t);
+                const y = point[1];
+
+                // Find points where it flips
+                if (
+                    // if there is an asymptote here, meaning that the graph
+                    // switches signs and has a large difference
+                    (y > 0 !== lastY > 0 &&
+                        Math.abs(y - lastY) >
+                            2 * this.drawingTransform().pixelsPerUnitY()) ||
+                    // or the function is undefined
+                    isNaN(y)
+                ) {
+                    // split the path at this point, and draw it
+                    paths.push(this.unstyledPath(points));
+                    // restart the path, excluding this point
+                    points = [];
+                } else {
+                    // otherwise, just add the point to the path
+                    // @ts-expect-error - TS2345 - Argument of type 'any' is not assignable to parameter of type 'never'.
+                    points.push(point);
+                }
+
+                lastY = y;
+            }
+
+            paths.push(this.unstyledPath(points));
+
+            return paths;
+        });
+    }
 
     plot(
         fn: (x: number) => number,
         range: Interval,
-        style?: Record<string, any>,
-    ): RaphaelElement {}
+        arg3?: boolean | Record<string, any>,
+        arg4?: Record<string, any>,
```

Comment:
We should have a method type for this similar to the one for Label. Otherwise you can pass two style objects.